.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "vitest.disableWorkspaceWarning": true
+}
```

Comment:
prefer not to change workspace settings - what is this for?

Reply:
I will remove